scripts/network_bandwidth.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+INTERVAL="1"  # update interval in seconds
```

Comment:
This should follow the standard that the other files follow. See the gpu_usage.sh file for a good example.